recipes/implot/all/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.4)
+project(implot CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+set(SOURCE_DIR source_subfolder)
+file(GLOB SOURCE_FILES ${SOURCE_DIR}/*.cpp)
```

Comment:
This globs a `demo.cpp`

Reply:
That's intended. ImGui series' demo cpp files are generally a part of their api, so that the library user can  visualize the apis. The `imgui` recipe also includes imgui_demo.cpp.